lib/std/priority_queue.zig

```diff
@@ -18,33 +18,26 @@ pub fn PriorityQueue(comptime T: type, comptime Context: type, comptime compareF
     return struct {
         const Self = @This();
 
-        items: []T,
-        cap: usize,
-        allocator: Allocator,
+        items: []T = undefined,
```

Comment:
default-initializing a field to `undefined` is a code smell. Almost always would be better with an init function or init value.

Reply:
Makes sense, I agree and as I mentioned in my other comment would normally do an `init()` function - I was just copying from [MultiArrayList](https://github.com/ziglang/zig/blob/41330c96aebddbf26d8fdc0725e7483476175601/lib/std/multi_array_list.zig#L22) in case that was somehow preferred. 

I'll change it to use an `init` function.